app/helpers/report_helper.rb

```diff
@@ -18,7 +18,7 @@ module ReportHelper
     :miq_rpt_gray_bg     => _("Gray Background")
   }.freeze
 
-  NOTHING_STRING = "<<< #{_('Nothing')} >>>".freeze
+  NOTHING_STRING = "#{_('<<<Nothing>>>')}".freeze
```

Comment:
Since this is a module constant, you need to use `N_()` here and use `_(NOTHING_STRING)` wherever the string is being rendered.